ui/intermittents-view/constants.js

```diff
@@ -0,0 +1,37 @@
+import * as d3 from "d3";
+
+// data is passed as a prop and target is updated via a ref in graph.jsx
+export const graphOneSpecs = {
+  title: "Orange Count per Push",
+  data: [],
+  width: 700,
+  height: 300,
+  right: 40,
+  interpolate: d3.curveLinear,
+  color: "#dd6602",
+  target: "",
+  x_accessor: "date",
+  y_accessor: "value"
+};
+
+export const graphTwoSpecs = {
+  data: [],
+  width: 700,
+  height: 300,
+  right: 40,
+  interpolate: d3.curveLinear,
+  color: ["blue", "green"],
+  target: "",
+  x_accessor: "date",
+  y_accessor: "value",
+  legend: ["Orange Count", "Push Count"],
+  legend_target: ".legend"
+};
+
+export const bugzillaDomain = "https://bugzilla.mozilla.org/";
```

Comment:
Could you move these and the URL like items in `ui/intermittents-view/helpers.js` to `ui/helpers/urlHelper.js`? :-)

Reply:
The last four items, but not the two specs right?

Reply:
No, just the URL-like helpers.